amr-wind/incflo_advance.cpp

```diff
@@ -397,8 +414,25 @@ void incflo::ApplyPredictor(bool incremental_projection)
     if (m_verbose > 2) {
         PrintMaxVelLocations("after nodal projection");
     }
+    // ScratchField to store the old np1
+    if (inonlin > 0) {
```

Comment:
I think it might be good to add a verbosity condition here:
```suggestion
    if (inonlin > 0 && m_verbose > 0) {
```